lib/vinyldns/api.rb

```diff
@@ -53,7 +53,7 @@ def self.make_request(signed_object, uri, body = '')
       url = URI(signed_object.api_url)
       https = Net::HTTP.new(url.host, url.port)
       https.use_ssl = true ? url.scheme == "https" : https.use_ssl = false
-      https.verify_mode = OpenSSL::SSL::VERIFY_NONE # SSL not signed? Don't care!
+      https.verify_mode = ENV['VINYLDNS_VERIFY_SSL'] == false || !ENV['VINYLDNS_VERIFY_SSL'].nil? ? OpenSSL::SSL::VERIFY_NONE : OpenSSL::SSL::VERIFY_PEER
```

Comment:
This will set `OpenSSL::SSL::VERIFY_NONE` if `ENV['VINYLDNS_VERIFY_SSL']` is _any_ value other than `false` or `nil`. To me, that seems problematic, as `export VINYLDNS_VERIFY_SSL=true` will _disable_ SSL verification and set `OpenSSL::SSL::VERIFY_NONE`.

Also, an OS environment variable set via a shell such as bash via a technique like `export VINYLDNS_VERIFY_SSL=false` will not be evaluated by Ruby as `false`, as I sought to demonstrate in my examples [here](https://github.com/vinyldns/vinyldns-ruby/pull/9#issuecomment-450645858). In Ruby, you can do `ENV['VINYLDNS_VERIFY_SSL'] = false` to set the variable to a boolean value, but I think it's far more likely that the user will establish the `VINYLDNS_VERIFY_SSL` environment variable via something akin to `export VINYLDNS_VERIFY_SSL=false` in bash or some other shell, which sets the variable to the string "false" and not the boolean `false`.